libs/tailwindcss-config/src/vega-custom-classes.js

```diff
@@ -15,14 +15,18 @@ const vegaCustomClasses = plugin(function ({ addUtilities }) {
     },
     '.syntax-highlighter-wrapper .hljs': {
       fontSize: '1rem',
-      fontFamily: "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace",
+      fontFamily: "'Roboto Mono', monospace",
```

Comment:
**nitpick (non-blocking)** : 🤨 :wink:

Reply:
Tailwind's `font-mono` is basically that ⬆️, no reason to mix fonts.